web/src/components/common/DatePicker.vue

```diff
@@ -47,7 +50,10 @@ export default {
         return true
       },
     },
-    label: String,
+    label: {
+      type: String,
+      default: undefined,
```

Comment:
Maybe label should be a required field if default is not provided?
Alternative set default label to "Pick a Date" or something (unless Vuetify provides it's own default).